packages/sourcecred/src/api/main/grain.js

```diff
@@ -24,8 +25,15 @@ export type GrainOutput = {|
   +ledger: Ledger,
 |};
 
+// Similar to GrainIntegrationResult but excludes the ledger
+// since only the most recent ledger is relevant
+export type GrainIntegrationMultiResult = {|
```

Comment:
nit: Why multi? Makes me think it's going to be an array. Not sure I have a better suggestion, I'm all turned around on the namings in this epic lol. timeStampedGrainIntegrationOutput? idk. you can keep it.